llvm/lib/Target/AMDGPU/VOP2Instructions.td

```diff
@@ -2520,16 +2520,22 @@ multiclass VOP2_Real_DOT_ACC_gfx10<bits<6> op> :
   VOP2_Real_dpp_gfx10<op>,
   VOP2_Real_dpp8_gfx10<op>;
 
+multiclass VOP2Only_Real_DOT_ACC_gfx10<bits<6> op> {
+  let IsSingle = 1 in
+    defm NAME : VOP2_Real_e32_gfx10<op>;
+  defm NAME : VOP2_Real_dpp_gfx10<op>, VOP2_Real_dpp8_gfx10<op>;
```

Comment:
I know this looks weird as we define two `NAME` here. Basically I want to set `IsSingle=1` for `VOP2_Real_DOT_ACC_gfx10`, but since `IsSingle` is only a valid field in `VOP2_Real_e32_gfx10`, I can only split them into two `defm`s. I'm new to TableGen so I'd like to learn what I need to do in this case.